app/templates/components/share-popup.hbs

```diff
@@ -0,0 +1,2 @@
+<i class="fa fa-share-square-o" aria-hidden="true"></i>
+<a id="share-popover" data-html="true" rel="popover" data-placement="top" data-content='<div><a><i aria-hidden="true" class="fa fa-twitter-square"></i>Tweet</a><br><a><i aria-hidden="true" class="fa fa-facebook-square"></i>Post to Facebook</a><br><a><i aria-hidden="true" class="fa fa-linkedin-square"></i>Share on LinkedIn</a><br><a><i aria-hidden="true" class="fa fa-envelope-square"></i>Send in email</a><br></div>' data-original-title="Share">Share</a>
```

Comment:
Making these links functional will be a next step but not necessary for this PR.